src/core/src/core_logic/PatchInstaller.py

```diff
@@ -176,14 +177,49 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
         self.last_still_needed_packages = all_packages
         self.last_still_needed_package_versions = all_package_versions
 
+        stopwatch_for_batch_install_process = Stopwatch(self.env_layer, self.telemetry_writer, self.composite_logger)
+        stopwatch_for_batch_install_process.start()
+
+        installed_update_count, patch_installation_successful, maintenance_window_batch_cutoff_reached = self.install_packages_in_batches(all_packages, packages, package_versions, maintenance_window, package_manager)
+
+        stopwatch_for_batch_install_process.stop()
+
+        # Check for packages available and install them in batches. Some packages may not be installed due to:
```

Comment:
nit: "Check for packages available and install them in batches." applies to line 183, so move it to line 182 or change this to "self.install_packages_in_batches checks and installs packages in batches, but some packages may not be installed due to"

Reply:
Done.